src/pages/learn/serving-over-http.mdx

```diff
@@ -1,22 +1,65 @@
+
+After authentication, the server should not make any [authorization](/learn/authorization/) decisions about the request until GraphQL execution begins. Specifically, field-level authorization should be done by the GraphQL schema during [GraphQL's ExecuteRequest()](https://spec.graphql.org/draft/#ExecuteRequest()), allowing for a partial response to be generated if authorization-related errors are raised.
+
+## Request format
 
-## Web Request Pipeline
+### Headers
 
-Most modern web frameworks use a pipeline model where requests are passed through a stack of middleware (AKA filters/plugins). As the request flows through the pipeline, it can be inspected, transformed, modified, or terminated with a response. GraphQL should be placed after all authentication middleware, so that you have access to the same session and user information you would in your HTTP endpoint handlers.
+GraphQL clients and servers should support JSON for serialization and may support other formats. Clients should also indicate what media types they support in responses using the `Accept` HTTP header. Specifically, the client should include the `application/graphql-response+json` in the `Accept` header.
 
-## URIs, Routes
+If no encoding information is included with this header, then it will be assumed to be `utf-8`. However, the server may respond with an error if a client doesn't provide the `Accept` header with a request.
+
+<Callout type="info">
+The `application/graphql-response+json` is described in the draft GraphQL over HTTP specification. To ensure compatibility, if a client sends a request to a legacy GraphQL server before 1st January 2025, the `Accept` header should also include the `application/json` media type as follows: `application/graphql-response+json; charset=utf-8, application/json; charset=utf-8`
+</Callout>
+
+### Methods
+
+Your GraphQL HTTP server must handle the HTTP `POST` method, and may also accept the `GET` method for incoming API requests.
+
+#### `POST` request and body
+
+A standard GraphQL POST request should set `application/json` for its `Content-type` header, and include a JSON-encoded body of the following form:
+
+```json
+{
+  "query": "...",
+  "operationName": "...",
+  "variables": { "myVariable": "someValue", ... },
+  "extensions": { "myExtension": "someValue", ... }
+}
+```
 
-HTTP is commonly associated with REST, which uses "resources" as its core concept. In contrast, GraphQL's conceptual model is an entity graph. As a result, entities in GraphQL are not identified by URLs. Instead, a GraphQL server operates on a single URL/endpoint, usually `/graphql`, and all GraphQL requests for a given service should be directed at this endpoint.
+The `query` parameter is required and will contain the source text of the GraphQL document. Note that this parameter may contain documents for query or mutation operations.
```

Comment:
Technically the document can contain `subscription` operations too, we just can't execute them. This is useful for when the document contains all of the operations of your application (a mixture of all three operation types), which some people do when using a single trusted document for the entirety of a mobile application.

```suggestion
The `query` parameter is required and will contain the source text of a GraphQL document. Note that the term `query` here is a misnomer: the document may contain any valid GraphQL operations (and associated fragments).
```